metadata-ingestion/tests/integration/dbt/dbt_test_column_meta_mapping_golden.json

```diff
@@ -129,7 +130,8 @@
         "json": {
             "typeNames": [
                 "Model",
-                "Table"
+                "Table",
```

Comment:
Something can have all these different subtypes at once?? Whoa.. What's expected in the UI?

Reply:
Yep - the UI shows the first one, but takes all of them into account when deciding whether or not to show the "View Properties" tab